doc/man3/SSL_CTX_set1_curves.pod

```diff
@@ -75,17 +75,76 @@ to downgrade connections to a weaker algorithm.
 =back
 
 SSL_CTX_set1_groups_list() sets the supported groups for B<ctx> to
-string B<list>. The string is a colon separated list of group names, for example
-"P-521:P-384:P-256:X25519:ffdhe2048". The groups are used as in
+string B<list>. As an extension to SSL_CTX_set1_groups(), the names of the
+groups rather than their NIDs are used. The groups are used as in
 SSL_CTX_set1_groups(), described above. Currently supported groups for
 B<TLSv1.3> are B<P-256>, B<P-384>, B<P-521>, B<X25519>, B<X448>,
 B<brainpoolP256r1tls13>, B<brainpoolP384r1tls13>, B<brainpoolP512r1tls13>,
 B<ffdhe2048>, B<ffdhe3072>, B<ffdhe4096>, B<ffdhe6144> and B<ffdhe8192>. Support
-for other groups may be added by external providers, however note the discussion
-on TLS 1.3 selection criteria above. If a group name is preceded with the C<?>
-character, it will be ignored if an implementation is missing. The "DEFAULT"
-group name is used to select the default selection of groups. Group names that
-are preceded with the C<-> character will be removed from the selected groups.
+for other groups may be added by external providers.
+The group can be either the B<NIST> name (e.g. B<P-256>), some other commonly
+used name where applicable (e.g. B<X25519>, B<ffdhe2048>) or an OpenSSL OID name
+(e.g. B<prime256v1>). Group names are case sensitive.
+
+The B<list> can be used to define several group tuples of comparable security
+levels, influence the number of key shares sent by a client, ignore unknown
+groups, add the OpenSSL default set of groups or allows to remove a group from
+the list.
```

Comment:
```suggestion
The B<list> can be used to define several group tuples of comparable security
levels, and can specify which key shares should be sent by a client.
The specified list elements can optionally be ignored, if not implemented
(listing unknown groups otherwise results in error).
It is also possible to specify the built-in default set of groups, and to explicitly
remove a group from that list.
```